### ISSUE TITLE
Beta Issue - Unable to delete type item

### ISSUE THREAD

**Issue by fredlef** [11cafe/comfyui-workspace-manager#153]
In the beta version the top item in the list of workflows cannot be moved to trash.  The trash icon changes to open in new tab icon when pointer is placed on it.

**Comment by Weixuanf** [comment]
oh weird, i can delete it with no problem, what browser are you using? https://youtu.be/4C3-CTm4Pqk

**Comment by fredlef** [comment]
I'm in Safari on a M3 Macbook

**Comment by fredlef** [comment]
Just retested in Chrome - ok, back to Safari - now ok.  I'll update if it recurs.

**Comment by Weixuanf** [comment]
oh ok, i just tested safari too, it's prob hided behind the scroll bar.
<img width="623" alt="Screenshot 2024-01-26 at 10 41 25 PM" src="https://github.com/11cafe/comfyui-workspace-manager/assets/18367033/7db044e7-bfeb-456e-b8b9-01a60732a461">
in general i would suggest to use chrome, bc we mostly test in chrome

**Comment by fredlef** [comment]
Ok will do - I'll keep an eye out for that.

Really like what I've seen so far.  Is the gallery icon being used - opens up a blank page?  I didn't see that accessed in the youtube video.

Thanks,

**Comment by Weixuanf** [comment]
Oh the gallery will only show images when you have generated images using this workflow. Try generating a images with current workflow and click the gallery icon then you should see images in it

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Weixuanf** [comment]
Oh great! I'm glad you see beta working well. Did you upgraded from the main branch (launched version)? Is the upgrading ok? (No data loss or anything?)

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by fredlef** [comment]
Yes - Upgraded from the main branch - worked well - no issues.   One interesting thing and not sure that it's of any concern but I did start using Chrome.  Not all of the workflows that I could see in Safari were evident in Chrome even though they were pointing to the same folder.  I imported the couple that were missing and it was working fine.

**Comment by fredlef** [comment]
I cannot get the gallery to populate under any circumstances.

Thanks

**Comment by Weixuanf** [comment]
Ohh so after upgrading, the image gallery doesn’t show any images anymore (old images and one ones))? Even when you generated new image and is it will not adding into the gallery?

And one thing to note about new version, the workflow data is stored in your browser storage so if you switch browser (chrome to safari) you may not see all of your workflows. So it's better to stick to one browser only. This design is to reduce the python network calls so read/write will not occupy bandwidth and improve performance.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by fredlef** [comment]
I think the gallery issue is related to MacBook.  Did not work before or after upgrade.  Running the beta on my windows laptop gallery worked before and after upgrade.  (Edge browser).

**Comment by Weixuanf** [comment]
That’s weird. I’m using Mac and the gallery is ok in my MacBook, are you using Save Image node to save the image to your output folder? It can only load images saved to your ComfyUI output folder.

results from preview nodes like Preview Image will not be loaded. Because it’s saving into temp folder and those images will disappear

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by fredlef** [comment]
That must have been it.  I added a save node and it's working properly.  By the way for sometime I have been wishing there was a away to save the prompt with an image so I really like the gallery - any chance of a folder structure?

**Comment by Weixuanf** [comment]
Great glad to know you like the new feature! 🥳
You mean you hope to add folders inside the gallery to organize the images? Since these images are stored in your output folder and to a bit hard to manipulate them to folders.
 but maybe we can enable tags or sub albums for organizing images which don't involve direct manipulation of your disk folders

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by fredlef** [comment]
Yes that's right.  Or maybe add a field for image tags that is searchable.  Or make all fields searchable.  Or "show me all pictures from X model" or show me all pictures that have "xxxxx xxxx" string in the prompt.
